### PR TITLE
Update CIs

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,9 +16,10 @@ script:
   - cargo test --features embed-lgpl-docs --jobs 1
   # catch any sneaked in lgpl docs
   - cargo build --features purge-lgpl-docs --jobs 1
+  - cargo doc --features "dox"
```
Comment (EPashkin) [inline .travis.yml]: Also need one line down.

File: appveyor.yml [hunk 2]
```diff
@@ -22,5 +22,6 @@ build_script:
   - cd _examples
   - cargo build
   - cargo build --features gtk_3_18
+  - cargo doc --features "dox"
```
Comment (EPashkin) [inline appveyor.yml]: and here need be in beginning build script

Comment (EPashkin) [reply appveyor.yml]: or need `cd ..`

**Comment by GuillaumeGomez** [comment]
This failure is **really** weird...

**Comment by EPashkin** [comment]
Seems there no https://github.com/gtk-rs/gio/blob/master/src/unix_socket_address.rs#L6-L7 for windows.
Currently I don't have access to docs so cant check, have windows `OsStrExt` or not

**Comment by EPashkin** [comment]
Understand, it really strange.
Gio documented for me locally, but I can't check gdk due bad access to crates.

**Comment by GuillaumeGomez** [comment]
I'll try to figure this out later.

**Comment by EPashkin** [comment]
@GuillaumeGomez 
Problem that OsExt::from_bytes is unix only.
This patch in gio fixes problem, but it sure wrong way to fix.
Also I still don't understand why docs builds for gio.
```diff
--- a/src/unix_socket_address.rs
+++ b/src/unix_socket_address.rs
@@ -78,7 +78,10 @@ impl<O: IsA<UnixSocketAddress> + IsA<glib::object::Object>> UnixSocketAddressExt
         };
         match self.get_address_type() {
             UnixSocketAddressType::Anonymous => Some(Anonymous),
+            #[cfg(unix)]
             UnixSocketAddressType::Path => Some(Path(path::Path::new(OsStr::from_bytes(path)))),
+            #[cfg(not(unix))]
+            UnixSocketAddressType::Path => panic!("Unsupported UnixSocketAddressType::Path"),
             UnixSocketAddressType::Abstract => Some(Abstract(path)),
             UnixSocketAddressType::AbstractPadded => Some(AbstractPadded(path)),
             UnixSocketAddressType::Invalid | UnixSocketAddressType::__Unknown(_) => None,
```

**Comment by GuillaumeGomez** [comment]
It's done recursively by `cargo doc` I assume. And yes, not a very beautiful way to do it. :)

**Comment by EPashkin** [comment]
As alternative we can add trait and impl for dox:
```rust
#[cfg(all(not(unix), feature = "dox"))]
trait OsStrExt {
    fn from_bytes(slice: &[u8]) -> &Self;
}

#[cfg(all(not(unix), feature = "dox"))]
impl OsStrExt for OsStr {
    fn from_bytes(_slice: &[u8]) -> &OsStr {
        unreachable!()
    }
}
```

**Comment by EPashkin** [comment]
Currently I prefer slightly changed first variant, it IMHO make intentions clearer
```rust
#[cfg(not(dox))]
UnixSocketAddressType::Path => Some(Path(path::Path::new(OsStr::from_bytes(path)))),
#[cfg(dox)]
UnixSocketAddressType::Path => unreachable!(),
```

**Comment by GuillaumeGomez** [comment]
I also do so let's go for it!

**Comment by EPashkin** [comment]
Added https://github.com/gtk-rs/gio/pull/115

**Pull request merged**